spec/features/filter_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+require 'features/pagination_spec_helper'
+
+RSpec.feature "Filter in content items", type: :feature do
+  context "filtering by search term" do
+    scenario "the user enters a text in the search box and retrieves a filtered list" do
+      create :content_item, title: "some text"
+      create :content_item, title: "another text"
+
+      visit "/content_items"
+      fill_in 'query', with: 'some text'
+
+      click_on "Filter"
```

Comment:
Very minor: Could you please check this file and move `click_on "Filter"` 2 lines above the expectation, and next to the user interactions